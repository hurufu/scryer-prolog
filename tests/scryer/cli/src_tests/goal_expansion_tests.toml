args = [
    "-f",
    "--no-add-history",
    "-g", "use_module(library(goal_expansion_tests))",
    "-g", "use_module(library(clpz))",
    "-g", "use_module(library(format))",
    "-g", "use_module(library(lambda))",
    "-g", "use_module(library(iso_ext))",
    "-g", "goal_expansion_tests:main(goal_expansion_tests)",
    "src/tests/goal_expansion/main.pl"
]
