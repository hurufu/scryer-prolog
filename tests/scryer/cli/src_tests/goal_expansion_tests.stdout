Running test "#2361 1"
Running test "#2361 2"
Running test "#2361 3"
Running test "#2361 4"
Running test "#2236 1"
Running test "#2236 2"
Running test "#2255 1"
Running test "#2255 2"
Running test "#2255 3"
Running test "#2255 4"
Running test "#2255 5"
Running test "#2255 6"
Running test "#2202 1"
Running test "#2062 1"
Running test "#2062 2"
Running test "#2062 3"
Running test "#2062 4"
Running test "#1977 1"
Running test "#1977 2"
Running test "#1977 3"
Running test "#1987 1"
